timetracker/entries/admin.py

```diff
@@ -6,7 +6,8 @@
 class EntryAdmin(admin.ModelAdmin):
     list_display = ('project', 'description', 'start', 'stop')
     list_filter = ('project',)
-    search_fields = ('project', 'description',)
```

Comment:
The reason we leave this in there is to ensure that if we ever remove one of the fields we don't accidentally leave it as:

    search_fields = ('project')

Which gets interpreted by Python as a string, not a tuple.

    >>> type(('project'))
    <type 'str'>
    >>> type(('project',))
    <type 'tuple'>

(just noticing that _we_ left it out of the `list_display` tuple)